ui/revenuecatui/src/test/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/processed/VariableProcessorTest.kt

```diff
@@ -173,7 +173,21 @@ class VariableProcessorTest {
 
     @Test
     fun `process variables processes sub_offer_duration`() {
-        expectVariablesResult("{{ sub_offer_duration }}", "INT_OFFER_DURATION")
+        expectVariablesResult("{{ sub_offer_duration }}", "1 month")
+        expectVariablesResult("{{ sub_offer_duration }}", "1 month", rcPackage = TestData.Packages.bimonthly)
+        expectVariablesResult("{{ sub_offer_duration }}", "2 weeks", rcPackage = TestData.Packages.quarterly)
+    }
+
+    @Test
+    fun `process variables processes sub_offer_duration for spanish locale`() {
+        expectVariablesResult("{{ sub_offer_duration }}", "1 mes", esLocale)
```

Comment:
Relying on the default package is confusing here, maybe always pass an explicit value?

Reply:
I had to look up the default value yes, maybe making it more explicit is better

Reply:
I added the default value in the tests where we test a bunch of packages in the followup PR: https://github.com/RevenueCat/purchases-android/pull/1285

I thought about making the parameter not optional, but then we have to specify the package in a bunch of extra tests which seemed like a lot of repetition.